tests/e2e/rudder-cdn-js/src/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+const root = ReactDOM.createRoot(
+  document.getElementById('root') as HTMLElement
+);
+root.render(
```

Comment:
can we place a snapshot in the description of the PR of how the sample website/page looks

Reply:
done